device-types/Supermicro/SuperServer-4029GP-TRT.yaml

```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Supermicro
+model: SuperServer 4029GP-TRT
+slug: supermicro-sys-4029gp-trt
+part_number: SYS-4029GP-TRT
+u_height: 4
+is_full_depth: true
+weight: 45.3
+weight_unit: kg
+airflow: front-to-rear
+comments: 'Specification: [https://www.supermicro.com/en/products/system/4u/4029/sys-4029gp-trt.cfm](https://www.supermicro.com/en/products/system/4u/4029/sys-4029gp-trt.cfm)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 2000
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 2000
+  - name: PSU3
+    type: iec-60320-c14
+    maximum_draw: 2000
+  - name: PSU4
+    type: iec-60320-c14
+    maximum_draw: 2000
```

Comment:
These PSUs appear to be modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder